src/library/methods/firebase.js

```diff
@@ -0,0 +1,18 @@
+export const createUser = () => {
+  auth()
+    .createUserWithEmailAndPassword('jane.doe@example.com', 'SuperSecretPassword!')
```

Comment:
Even if it's an example, you guys try not to upload hard data to GitHub. You can make use of parameters.

```suggestion
export const createUser = (emailUser,passwordUser) => {
  auth()
    .createUserWithEmailAndPassword(emailUser,passwordUser)
```

Reply:
Nice, parameters have been added.